geomstats/visualization/beta_distributions.py

```diff
@@ -0,0 +1,426 @@
+"""Visualization for Geometric Statistics."""
+import matplotlib.pyplot as plt
+
+import geomstats.backend as gs
+from geomstats.geometry.special_euclidean import SpecialEuclidean
+from geomstats.information_geometry.beta import BetaDistributions
+
+SE2_GROUP = SpecialEuclidean(n=2, point_type="matrix")
+
+beta = BetaDistributions()
+
+
+class Beta:
+    """Class for the visualization of beta manifold."""
+
+    def process_points(self, points, **kwargs):
+        """Process all manifold points.
+
+        Confirms that points passed into function lie on manifold.
+        Prepares points for plotting.
+
+        by Marianne Arriola
+
+        Parameters
+        ----------
+        points : array-like, shape=[..., 2]
+            Beta manifold points to be plotted.
+        """
+        points = gs.array(points)
+        if len(points.shape) == 1:
+            points = gs.expand_dims(points, axis=0)
+
+        if not len(points) > 0:
+            raise ValueError("No points given")
+        if not gs.all(points > 0):
+            raise ValueError(
+                "Points must be in the upper-right quadrant of Euclidean space"
+            )
+        if not ((points.shape[-1] == 2 and len(points.shape) == 2)):
+            raise ValueError("Points must lie in 2D space")
+        limit = gs.amax(points)
+        limit += limit / 10
+        return points, limit
+
+    def plot(self, points, size=None, **kwargs):
+        """Draws the beta manifold.
```

Comment:
Docstring's oneliner needs to be in imperativ form: Draws ----> Draw
change in other docstrings as well